overops-java-bullshifier/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for overops-java-bullshifier.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Overops Config settings
+overops:
+  collectorHost: ""
+  collectorPort: 6060
+  applicationName: ""
+  deploymentName: ""
+  serverName: ""
+  color: "white"
+  noAgent: false # Use for tests
+
+  # Use an agent-sidecard instead of an embedded v4 agent (https://github.com/takipi-field/kubernetes/tree/master/agent/sidecar).
+  agentSidecar:
+    enabled: false
+    useDebug: false
+    repository: overops/agent-sidecar
+    pullPolicy: IfNotPresent
+    tag: latest
+
+  # Job Related settings
+  job:
+    enabled: false
+    runnindDurationHours: 0
+    runnindDurationMinutes: 15
+
+replicaCount: 1
+
+image:
+  repository: overops-java-bullshifier-v4
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
```

Comment:
Both the service and ingress sections can be removed as the bullishifier doesn't have inbound connections.

Reply:
Done